lib/std/zig/ast.zig

```diff
@@ -584,7 +584,7 @@ pub const Node = struct {
     }
 
     pub const Root = struct {
-        base: Node = Node {.id = .Root},
+        base: Node = Node{ .id = .Root },
```

Comment:
Why all these changes? but also, could use: `base: Node = .{.id = .Root}` or `base = Node{.id = .Root}`: having `Node` twice is redundant.

Reply:
> Why all these changes? 

See https://github.com/ziglang/zig/pull/3906#discussion_r357930745

Not having `Node` twice would be ideal but it doesn't currently work due to #3672. `base = Node{.id = .Root}` also doesn't work because the compiler can't infer field type from default value.

Reply:
doh. I didn't realise this was inside of a struct definition and that they were default fields.
/me stupid